src/pages/surveys/createSurvery/index.tsx

```diff
@@ -0,0 +1,200 @@
+import { BorderRadius, Button, ButtonSize } from '@/components/button';
+import { Card } from '@/components/card';
+import { Input } from '@/components/input';
+import { PanelNavigationItem } from '@/components/Panel';
+import { RoleOption } from '@/components/profileForm';
+import { SearchSelect } from '@/components/searchSelect';
+import { CircledPlus, Photo, SmallBin } from '@/components/svg/surveys/Surveys';
+
+function CreateSurvey({}) {
+  const SurveyQuestions: PanelNavigationItem[] = [
+    { title: 'Question', id: 'question ' },
+    { title: 'Option format', id: 'option format' },
+  ];
+
+  const OtherList: PanelNavigationItem[] = [
+    { title: 'Duration of survey', id: 'duration of survey ' },
+    { title: 'Points awarded (optional)', id: 'points awarded (optional)' },
+  ];
+
+  const Options: RoleOption[] = [
+    { label: '5 Minutes', value: '5 Minutes' },
+    { label: '10 Minutes', value: '10 Minutes' },
+    { label: '15 Minutes', value: '15 Minutes' },
+  ];
+
+  const AwardOptions: RoleOption[] = [
+    { label: '20 Points', value: '20 points' },
+    { label: '30 Points', value: '30 points' },
+    { label: '40 Points', value: '40 points' },
+  ];
+
+  return (
+    <div className='grid'>
+      <Card
+        hasHeader
+        hasBadge
+        hasBorder
+        className='!border-b-light-secondary-light_blue'
+        title='Create new survey'
+      >
+        <>
+          <div className='max-w-[620px] mt-10 mb-2'>
+            <p className='font-medium text-3xl text-light-grey-200 pb-2'>
+              Title of the survey
+            </p>
+            <Input
+              placeHolder='Enter title of the survey'
+              isCurved
+              hasBorder
+              className='!border-light-blue-50 !drop-shadow-none placeholder:text-[#1C1C1E]'
+            />
+          </div>
+        </>
+      </Card>
+
+      <Card
+        hasHeader
+        hasBadge
+        hasBorder
+        className='!border-b-light-secondary-light_blue'
+        title='Add survey question'
+      >
+        <div className=' relative grid grid-cols-[minmax(250px,480px)_minmax(250px,480px)] justify-between gap-5 pb-10'>
+          {SurveyQuestions.map((item) => (
+            <>
+              <div className='max-w-[620px] mt-10 mb-2'>
+                <p className='font-medium text-3xl text-light-grey-200 pb-2'>
+                  {item.title}
+                </p>
+                {item.id === 'option format' ? (
+                  <Input
+                    placeHolder='Multiple Select'
+                    isCurved
+                    hasBorder
```

Comment:
You can see what i mean in the last UI for survey they used check boxes as an example of one of the options so do justice to it @Maap014